src/einsteinpy/misc/dual.py

```diff
@@ -0,0 +1,222 @@
+import numpy as np
+
+
+class Dual:
```

Comment:
A better and more descriptive class name would be appreciated

Reply:
True. `Dual` does seem like, it can be confused with a tensor operation. `DualNumber` should work, right?